### ISSUE TITLE
[lsoda] 500 steps taken before reaching tout

### ISSUE THREAD

**Issue by n-mourdou** [Nicholaswogan/numblsoda#14]
I am running a simulation and if I understand correctly the maximum number of steps (500) is performed before the integration finishes. Is there a way to change this parameter?

Thanks

**Comment by Nicholaswogan** [comment]
There might be a maximum number of steps between the evaluation times 't_eval'. This is something that could be adjusted. Can you give me more information about why you are asking.

**Comment by n-mourdou** [comment]
Sure. I am trying to solve a stiff ODE system, with a quite complicated right-hand side part. 

It would make sense that the message "[lsoda] 500 steps taken before reaching tout" refers to steps between times in 't_eval', as a Scipy solution that I have does indeed need more than 500 steps to integrate between some time points. The whole point is to speed this solution up by using numbaLsoda.

**Comment by Nicholaswogan** [comment]
Increasing max steps is not possible right now. I need to adjust the underlying C++, to expose that option to python.

**Comment by n-mourdou** [comment]
Got it. Thank you for the answer.

**Comment by Nicholaswogan** [comment]
Fixed with 5ad8cce583a2e1dfb721d7f4d6c9b135b718bd60

Now you can set with 

```python
usol, success = dop853(funcptr, u0, t_eval, data=data, rtol=1e-8, atol=1e-8,mxstep=1000)
```
